imagebuildah/build.go

```diff
@@ -527,6 +530,7 @@ func NewExecutor(store storage.Store, options BuildOptions) (*Executor, error) {
 		squash:                options.Squash,
 		labels:                append([]string{}, options.Labels...),
 		annotations:           append([]string{}, options.Annotations...),
+		onbuild:               append([]string{}, options.OnBuild...),
```

Comment:
I was able to comment this out and it seemed to work without it.

Reply:
These fields are necessary to get 

buildah config --onbuild "RUN echo foobar" $CTR

To work.